src/vue/pages/Funds.vue

```diff
@@ -1,61 +1,403 @@
 <template>
   <div class="funds">
     <top-bar>
+      <template slot="main">
+        <select-field
+          :disabled="!isLoaded"
+          v-model="filters.baseAsset"
+          :values="fundAssets"
+          class="app__select app__select--no-border"
+        />
+
+        <select-field
+          :disabled="!isLoaded"
+          v-model="filters.state"
+          :values="FUND_STATES"
+          class="app__select app__select--no-border"
+        />
+
+        <select-field
+          :disabled="!isLoaded"
+          v-model="filters.sort"
+          :values="FUND_SORT_STATES"
+          class="app__select app__select--no-border"
+        />
+      </template>
+
       <template slot="extra">
         <button
           v-ripple
-          v-if="account.accountTypeI === ACCOUNT_TYPES.syndicate"
           class="app__button-raised"
           @click="isCreateFundDrawerShown = true"
         >
           <i class="mdi mdi-plus funds__btn-icon" />
-          {{ 'funds.create-fund' | globalize }}
+          {{ 'funds.create-fund-title' | globalize }}
         </button>
       </template>
     </top-bar>
+
     <drawer :is-shown.sync="isCreateFundDrawerShown">
       <template slot="heading">
-        {{ 'create-fund-form.create-fund-header' | globalize }}
+        {{ 'funds.create-fund-title' | globalize }}
       </template>
-      <create-fund-form @close="isCreateFundDrawerShown = false" />
+      <create-fund-form @cancel="isCreateFundDrawerShown = false" />
     </drawer>
+
+    <drawer :is-shown.sync="isDetailsDrawerShown">
```

Comment:
let's consider moving it to the inner page

Reply:
Updated